cmd/goplugin-heroku/resource/app.go

```diff
@@ -0,0 +1,110 @@
+package resource
+
+import (
+	"context"
+	"os"
+
+	"github.com/hashicorp/go-hclog"
+	heroku "github.com/heroku/heroku-go/v5"
+	"github.com/lyraproj/servicesdk/serviceapi"
+)
+
+type App struct {
+	AppID        *string
+	Locked       *bool
+	Maintenance  *bool
+	Name         string
+	Organization *string
+	Personal     *bool
+	Region       string
+	Space        *string
+	Stack        string
+}
+
+type AppHandler struct{}
+
+func (*AppHandler) Create(desiredState *App) (*App, string, error) {
+	hclog.Default().Debug("Creating Heroku App", "desiredState", desiredState)
+
+	s := herokuService()
+
+	// Creates a resource and allocates an ID that can be used to read it in the future
+	app, err := s.OrganizationAppCreate(context.TODO(), heroku.OrganizationAppCreateOpts{
+		Locked:       desiredState.Locked,
+		Name:         &desiredState.Name,
+		Organization: desiredState.Organization,
+		Personal:     desiredState.Personal,
```

Comment:
The `Personal` attribute is problematic because it cannot be subsequently retrieved by a read. See #328 for possible solution.

Reply:
May have a similar problem with `Path` in `build.go` on line https://github.com/lyraproj/lyra/pull/313/files#diff-de40d7ec6f34bc2a1d332f56d2c58ddaR32, maybe more so, as that attribute is not directly used by Heroku's API at all, but is used in conjuction with other API endpoints to create a valid `URL`